src/pages/jeonyongmin/Login/Login.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { Link, withRouter } from 'react-router-dom';
+import './Login.scss';
+
+class Login extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      id: '',
+      password: '',
+      buttonColor: false,
+    };
+  }
+
+  goToMain = e => {
+    this.props.history.push('./mainym');
+  };
+
+  handleIdPwInput = e => {
+    if (e.target.type === 'text') {
+      this.setState({ id: e.target.value }, this.validation);
+    } else if (e.target.type === 'password') {
+      this.setState({ password: e.target.value }, this.validation);
+    }
+  };
+
+  validation = () => {
+    const { id, password } = this.state;
+    const condition = id.includes('@') && password.length >= 5;
+    this.setState({ buttonColor: condition });
+  };
+
+  render() {
+    const { id, password, buttonColor } = this.state;
+    const condtionOfButtonActivated = id.includes('@') && password.length >= 5;
+    return (
+      <main className="loginYM">
+        <div className="loginPage">
+          <h1 className="loginTitle">Westagram</h1>
+          <div className="loginContents">
+            <form action="#" method="get" className="loginInformation">
+              <input
+                className="loginInputId input"
+                type="text"
+                placeholder="전화번호, 사용자 이름 또는 이메일"
+                required
+                onChange={this.handleIdPwInput}
+              />
+              <input
+                className="loginInputPassword input"
+                type="password"
+                placeholder="비밀번호"
+                required
+                onChange={this.handleIdPwInput}
+              />
+              <button
+                className={
+                  buttonColor ? 'loginButtonActive' : 'loginButtonDeactive'
+                }
+                type="submit"
+                onClick={this.goToMain}
+                disabled={condtionOfButtonActivated ? false : true}
+              >
+                로그인
+              </button>
+            </form>
+          </div>
+          <a
+            href="https://www.instagram.com/accounts/password/reset/"
+            className="loginFindPassword"
+          >
+            비밀번호를 잊으셨나요?
+          </a>
```

Comment:
리액트에서는 효율적인 렌더링을 위해 a태그의 활용보다 Link태그를 권장한다고 합니다!
```suggestion
          <Link to="/https://www.instagram.com/accounts/password/reset/">
            className="loginFindPassword"
          >
            비밀번호를 잊으셨나요?
          </Link>
```

Reply:
확인했습니다.